projects/packages/videopress/src/client/block-editor/blocks/video/edit.tsx

```diff
@@ -514,11 +514,19 @@ export default function VideoPressEdit( {
 					isRequestingVideoData={ isRequestingVideoData }
 					{ ...{ attributes, setAttributes } }
 				/>
+
 				<PlaybackPanel { ...{ attributes, setAttributes, isRequestingVideoData } } />
+
 				<PrivacyAndRatingPanel
 					{ ...{ attributes, setAttributes, isRequestingVideoData, privateEnabledForSite } }
 				/>
-				<ColorPanel { ...{ attributes, setAttributes, isRequestingVideoData } } />
+			</InspectorControls>
+
+			<InspectorControls __experimentalGroup="color">
```

Comment:
Would it be worth adding an inline comment here to note when this deprecated `__experimentalGroup` can be replaced with the stabilized `group` prop? Maybe the comment could link to the [PR stabilizing that prop](https://github.com/WordPress/gutenberg/pull/47105/files#diff-f1d682ce5edd25698e5f189ac8267ab659d6a786260478307dc1352589419309) as well. What do you think?

Reply:
https://github.com/Automattic/jetpack/pull/29152 sorry I didn't lose the ReadyToMerge state :-)